common/endpoints/endpoints.go

```diff
@@ -32,6 +34,10 @@ type TwitterServer struct {
 	Handlers map[string]http.Handler
 }
 
+func (s *TwitterServer) String() string {
+	return fmt.Sprintf("TwitterServer:Addr:%s, Stats:%s, Handlers:%v", s.Addr, s.Stats, s.Handlers)
```

Comment:
stats.StatsReceiver doesn't have a String() method, so the %s won't be that helpful. Recommend using %+v instead (maybe same with the Handlers)